include/mbedtls/config.h

```diff
@@ -1350,6 +1350,19 @@
  */
 #define MBEDTLS_SSL_ALL_ALERT_MESSAGES
 
+/**
+ * \def MBEDTLS_SSL_RECORD_CHECKING
+ *
+ * Enable the API mbedtls_ssl_check_record() which allows to check the
```

Comment:
I think API can be one single function as well.

Reply:
I really don't agree with this use of the language. It might be commonly used in this way, but that doesn't mean it's right! Checking with the Tech Authors they agree. It's minor though, and on its own I don't think it merits rework.

However I think the sentence is also grammatically incorrect. The sentence is currently:
> Enable the API mbedtls_ssl_check_record() which allows to check the validity, freshness and authenticity of an incoming record without modifying the externally visible state of the SSL context.

I don't think you can follow 'which allows' with 'to check the validity'. I can't work out the mistake, but at a minimum it doesn't scan. 

I'd suggest the following:
> Enable the function mbedtls_ssl_check_record() which can be used to check the validity and authenticity of an incoming record, to verify that it has not been seen before, and to check it without modifying the externally visible state of the SSL context.

Reply:
Done